### DISCUSSION TITLE
Check if a pdf page has tables

### DISCUSSION THREAD

**Discussion by vignesh0710** [pymupdf/PyMuPDF#3322] [Looking for help]
Is it possible to check if a pdf page has tables
```

import fitz #1.22.1

doc = fitz.open("my_pdf.pdf")
page = doc[1]

table_list = page.extract_tables() # found this solution online and this does not work

if len(table_list) > 0:
    return True
```

**Comment by JorjMcKie** [comment]
Your code snippet was probably generated by an AI - and therefore wrong.

Since v. 1.23.0 PyMuPDF supports table extraction from document pages: `tabs= page.find_tables()`. For details look [here](https://pymupdf.readthedocs.io/en/latest/page.html#Page.find_tables).
